### PR TITLE
WNC, SST1-N1

### PR DESCRIPTION
**Pull request by vic1231**
WNC, SST1-N1 first version.

### CODE REVIEW

File: machine/wnc/wnc_sst1_n1/machine.make [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+# Set the desired uClibc version
+UCLIBC_VERSION = 0.9.33.2
+
+#
+# Console parameters can be defined here (default values are in
+# build-config/arch/x86_64.make).
+# For example,
+# 
+CONSOLE_SPEED = 57600
```
Comment (cbrune) [inline machine/wnc/wnc_sst1_n1/machine.make]: can this be 115200?  That is the ONIE standard baud rate.

Comment (vic1231) [inline machine/wnc/wnc_sst1_n1/machine.make]: Okay, we will modify our console speed to 115200 in BIOS FW and ONIE code.

**Comment by cbrune** [comment]
Looks good, thanks.  Pushed to master:

- f287942 Modify CONSOLE_SPEED to 115200
- fb4216e WNC, SST1-N1

**Comment by cbrune** [comment]
I should have said this early, but your commit message was a little brief.  For the good of all (and your own marketing purposes) it would be better to describe what a WNC-SST1-N1 is.  I have no idea what kind of switch it is.  How many ports does it have?  What kind of ASIC?  What speeds?  Copper, optics?

Also it is encouraged to put a short description of what the machine is (how many ports, what speed) into the INSTALL file.

**Comment by vic1231** [comment]
Thanks, Curt. I will update later.

2015-07-08 1:19 GMT+08:00 Curt Brune <notifications@github.com>:

**Pull request closed without merging**